src/routes/routes.ts

```diff
@@ -1,5 +1,5 @@
 export enum ArcRoutes {
   DASHBOARD = '/',
-  TRANSACTION = '/transaction/:ID',
-  TRANSACTIONS = '/transactions/'
+  TRANSACTION = 'transaction/:ID',
```

Comment:
I don't understand why drop the initial slash. Speaking in term of "standard web path" a ref with a initial slash says

> this path is possible only with a root element (ex domain/transaction/123

while without using a initial slash 

> this path is possibile wherever you want (ex domain/bla/bla/transaction/123)